### PR TITLE
Function names eh

### PR DESCRIPTION
**Pull request by ehinman**
Added "TADA_" to all function names and switched them to the format TADA_CapitalLetterForEachWord.

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -1,47 +1,46 @@
...
-export(JoinWQPProfiles)
-export(PotentialDuplicateRowID)
-export(QAPPDocAvailable)
-export(QAPPapproved)
-export(QualityControlActivity)
-export(TADABigdataRetrieval)
-export(TADAOverviewMap)
-export(TADAReadWQPWebServices)
+export(TADA_AboveNationalWQXUpperThreshold)
+export(TADA_AggregatedContinuousData)
```
Comment (cristinamullin) [inline NAMESPACE:5]: Change to TADA_FindContinuousData ?

Comment (ehinman) [reply NAMESPACE:5]: Sure! Works for me.

File: NAMESPACE [hunk 2]
```diff
@@ -1,47 +1,46 @@
...
-export(InvalidSpeciation)
-export(JoinWQPProfiles)
-export(PotentialDuplicateRowID)
-export(QAPPDocAvailable)
-export(QAPPapproved)
-export(QualityControlActivity)
-export(TADABigdataRetrieval)
-export(TADAOverviewMap)
-export(TADAReadWQPWebServices)
+export(TADA_AboveNationalWQXUpperThreshold)
```
Comment (cristinamullin) [inline NAMESPACE:4]: Should we shorten this and the lower threshold function name? e.g. TADA_AboveUpperThreshold

Comment (ehinman) [reply NAMESPACE:4]: I'm into it. Sounds good. And same with BelowLower.

File: NAMESPACE [hunk 3]
```diff
@@ -1,47 +1,46 @@
...
+export(TADA_FieldValuesPie)
+export(TADA_FieldValuesTable)
 export(TADA_FindPotentialDuplicates)
+export(TADA_GetActivityTypeRef)
+export(TADA_GetDetCondRef)
+export(TADA_GetDetLimitRef)
+export(TADA_GetMeasureUnitRef)
+export(TADA_GetTemplate)
+export(TADA_GetWQXCharValRef)
+export(TADA_HarmonizationRefTable)
```
Comment (cristinamullin) [inline NAMESPACE:25]: Consider changing to TADA_SynonymRefTable

File: NAMESPACE [hunk 4]
```diff
@@ -1,47 +1,46 @@
...
+export(TADA_HarmonizeData)
```
Comment (cristinamullin) [inline NAMESPACE:26]: Consider changing to TADA_HarmonizeSynonyms

**Comment by cristinamullin** [comment]
[crosswalk.xlsx](https://github.com/USEPA/TADA/files/11940390/crosswalk.xlsx)

**Review by cristinamullin — APPROVED** [review]

**Pull request merged**